src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -35,7 +35,7 @@ import {
     CNA_COLOR_AMP,
     CNA_COLOR_HOMDEL,
     DEFAULT_GREY,
-    MUT_COLOR_FUSION,
+    MUT_COLOR_STRUCTURAL_VARINAT,
```

Comment:
Is structural variant a mutation?

Reply:
I think so.

Reply:
@kalletlak I don't think we categorize SV under mutations any more. It should be similar to CNA. We could name it as STRUCTURAL_VARINAT_COLOR. In the future, we might want to include different colors for different SV. cc @jjgao